kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/sentry/_Utils.kt

```diff
@@ -72,14 +72,14 @@ public fun Scope.breadcrumb(
 }
 
 /** Convenience function for creating and testing a sub-transaction. **/
-public inline fun <T> ITransaction.transaction(name: String, operation: String, body: (ITransaction).() -> T) {
+public inline fun <T> transaction(name: String, operation: String, body: ITransaction.() -> T) {
```

Comment:
What's the impact of doing this? I feel like this was probably intentional, but I don't entirely recall.

Reply:
From what I can tell there isn't a change. Tests all complete fine and nothing changes about the type of the function based on weather the `ITransaction.` prefix is there or not

Reply:
Right, but I feel like I namespaced it for a reason - can you not think of one?

Reply:
I can't I'm afraid. I've re-added the receiver to avoid any potential issues.